newdocs/README.md

```diff
@@ -26,8 +26,4 @@ This command generates static content into the `build` directory and can be serv
 
 ### Deployment
```

Comment:
I was thinking about adding the Netlify deploy status badge to this README or the repo's README, WDYT?

Reply:
sure :+1: